.github/workflows/lint.yml

```diff
@@ -11,6 +11,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
+      - name: Install libpq 14
+        run: |
+          echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -c -s)-pgdg main 14" | sudo tee /etc/apt/sources.list.d/pgdg14.list
+          sudo apt-get -qq update
+          sudo apt-get -y install libpq-dev>=14 libpq5>=14
```

Comment:
Is libpq 14 needed in the lint pipeline?

Reply:
Possibly not. I think I added that before making bindings conditional on libpq version, and these checks failed otherwise.

Reply:
Removed